src/pages/Admin/VenueList/VenueList.tsx

```diff
@@ -28,7 +28,7 @@ const VenueList: React.FC<VenueListProps> = ({
         </Link>
       </div>
       <ul className="page-container-adminsidebar-venueslist">
-        {venues.map((venue, index) => (
+        {relatedVenues.map((venue, index) => (
           <li
             key={index}
```

Comment:
`relatedVenues` will always have the `id` available, so we may as well use it here instead of the antipattern of using the `index` as the `key`:
```suggestion
        {relatedVenues.map((venue) => (
          <li
            key={venue.id}
```